@dctrwatson the CI failed:
```
make govet
make[1]: Entering directory `/home/travis/gopath/src/github.com/cilium/cilium'
GO111MODULE=off go vet \
    ./api/... \
    ./bugtool/... \
    ./cilium/... \
    ./cilium-health/... \
    ./common/... \
    ./daemon/... \
    ./operator/... \
    ./pkg/... \
    ./plugins/... \
    ./proxylib/... \
    ./test/. \
    ./test/config/... \
    ./test/ginkgo-ext/... \
    ./test/helpers/... \
    ./test/runtime/... \
    ./test/k8sT/... \
    ./tools/...
# github.com/cilium/cilium/pkg/ipam
pkg/ipam/crd.go:245:28: n.conf.IPv4NativeRoutingCIDR undefined (type Configuration has no field or method IPv4NativeRoutingCIDR)
# github.com/cilium/cilium/pkg/ipam
vet: pkg/ipam/crd.go:245:29: n.conf.IPv4NativeRoutingCIDR undefined (type Configuration has no field or method IPv4NativeRoutingCIDR)
make[1]: *** [govet] Error 2
make[1]: Leaving directory `/home/travis/gopath/src/github.com/cilium/cilium'
make: *** [unit-tests] Error 2
The command "./.travis/build.sh" exited with 2.

```

--- a/pkg/ipam/ipam.go
+++ b/pkg/ipam/ipam.go
@@ -71,6 +71,10 @@ type Configuration interface {
 	// SetIPv4NativeRoutingCIDR is called by the IPAM module to announce
 	// the native IPv4 routing CIDR if it exists
 	SetIPv4NativeRoutingCIDR(cidr *cidr.CIDR)
+
+	// IPv4NativeRoutingCIDR is called by the IPAM module retrieve
+	// the native IPv4 routing CIDR if it exists
+	IPv4NativeRoutingCIDR() *cidr.CIDR
 }
 
 // Owner is the interface the owner of an IPAM allocator has to implement

--- a/pkg/ipam/ipam_test.go
+++ b/pkg/ipam/ipam_test.go
@@ -56,6 +56,7 @@ func (t *testConfiguration) HealthCheckingEnabled() bool              { return t
 func (t *testConfiguration) IPAMMode() string                         { return option.IPAMHostScopeLegacy }
 func (t *testConfiguration) BlacklistConflictingRoutesEnabled() bool  { return false }
 func (t *testConfiguration) SetIPv4NativeRoutingCIDR(cidr *cidr.CIDR) {}
+func (t *testConfiguration) IPv4NativeRoutingCIDR() *cidr.CIDR        { return nil }
 
 func (s *IPAMSuite) TestLock(c *C) {
 	fakeAddressing := fake.NewNodeAddressing()

